### DISCUSSION TITLE
Lost seam as is in Kisslicer and extra restart distance functions?

### DISCUSSION THREAD

**Discussion by RiMr71** [OrcaSlicer/OrcaSlicer#1831] [Ideas]
Hey,
the Orca slicer is great and I thank the authors for their work.
Anyway, I have two suggestions:
- a "lost" seam like Kisslicer has - simply the start and end of the path is on the inner perimeters, the seams are then really much less visible, even on cylindrical objects
- Extra restart distance feature (that's what it's called in Simplify3D) - almost a necessary feature for printing with foam filaments like PLA-LW, for models that don't print like a vase. This missing feature is the only reason I still use Simplify3D for printing aircraft models.
Thanks and happy printing!
Richard
![small extra restart distance](https://github.com/SoftFever/OrcaSlicer/assets/16500983/6f94c979-5b18-4441-943f-587108a04283)
![Zero extra restart distance](https://github.com/SoftFever/OrcaSlicer/assets/16500983/0be2fa03-c4db-4589-89c9-9d6cc072e9af)
![lost seam](https://github.com/SoftFever/OrcaSlicer/assets/16500983/c161c533-a270-45f6-84b8-876ac4c9afe0)

**Comment by DogHudson** [comment]
Yes, I 2nd this request please! 

Kisslicer can produce practically seamless prints after you run the wizards properly. I'm surprised other slicers haven't tried to incorporate the tucked-in seam method.
